mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule DigitalSignature.Mixfile do
   def application do
     [extra_applications: [:logger, :confex, :runtime_tools, :poison,
                           :cowboy, :httpoison, :ecto, :ecto_paging, :postgrex,
-                          :phoenix,
+                          :phoenix, :phoenix_pubsub,
```

Comment:
Do we need to add `:phoenix_pubsub`

Reply:
Distillery gives some warning that `phoenix_pubsub` can't be found without adding this to `extra_applications`